File: explorer/src/main/java/com/duongame/listener/TextOnTouchListener.java
package com.duongame.listener;

import com.duongame.activity.TextActivity;

/**
 * Created by namjungsoo on 2017-01-22.
 */

public class TextOnTouchListener extends BaseOnTouchListener {
    private TextActivity activity;

    public TextOnTouchListener(TextActivity activity) {
        super(activity);
        this.activity = activity;
        touchAxis = Axis.AXIS_Y;
    }

    @Override
    protected boolean handleActionUp() {
        if (!isBeingDragged) {
            activity.setFullscreen(!activity.getFullscreen());
            return true;
        } else {
            isBeingDragged = false;
        }
        return false;
    }
}

File: Explorer/src/main/java/com/duongame/listener/TextOnTouchListener.kt
package com.duongame.listener

import com.duongame.activity.viewer.TextActivity
import com.duongame.listener.BaseOnTouchListener

/**
 * Created by namjungsoo on 2017-01-22.
 */
class TextOnTouchListener(private val activity: TextActivity) : BaseOnTouchListener(
    activity
) {
    override fun handleActionUp(): Boolean {
        if (!isBeingDragged) {
            activity.isFullscreen = !activity.isFullscreen
            return true
        } else {
            isBeingDragged = false
        }
        return false
    }

    init {
        touchAxis = Axis.AXIS_Y
    }
}